devel/cquery/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+PortGroup       waf 1.0
+PortGroup       xcodeversion 1.0
+
+name            cquery
+
+categories      devel
+license         MIT
+maintainers     {wyuenho @wyuenho} openmaintainer
+
+description     C/C++/Objective C Language Server Protocol support
+
+long_description \
+    cquery is a highly-scalable, low-latency language server for \
+    C/C++/Objective-C. It is tested and designed for large code bases like \
+    Chromium. cquery provides accurate and fast semantic analysis without \
+    interrupting workflow. \
+    \
+    cquery implements almost the entire language server protocol and provides some \
+    extra features.
+
+platforms       darwin
+os.version      16.0.0
```

Comment:
Why? All of these variables (including `os.version`) except `prefix` are read-only! https://guide.macports.org/#reference.variables

Reply:
Sorry, I don't know what I'm doing. I don't speak Tcl. I'll make a quick fix.

Reply:
BTW, why doesn't the linter catches that?

Reply:
Because nobody has implemented a check for this, anyone can implement it and open a pull request in [base](https://github.com/macports/macports-base).